pyface/tasks/advanced_editor_area_pane.py

```diff
@@ -7,7 +7,12 @@
 # is also available online at http://www.enthought.com/licenses/BSD.txt
 #
 # Thanks for using Enthought open source!
-# Import the toolkit specific version.
+
+""" Toolkit-specific implementation of the ``AdvancedEditorAreaPane``.
```

Comment:
Usually when "toolkit-specific" is mentioned, I expect a specific toolkit being associated with it, e.g. Foo is a Qt-specific implementation of Bar.

The original comment is fine as in it is importing the Qt-specific or the Wx-specific AdvancedEditorAreaPane, but it is commenting about the implementation details. The public API of this module should not need to worry about that. AdvancedEditorAreaPane itself should have a toolkit independent API. So the sentence reads erroneously alarming to me: It is toolkit-specific, so I need to be using a particular toolkit in order to use this class, what is that toolkit?

Perhaps we can just remove this sentence unless we have a better replacement for it?

(Likewise for the other occurrences.)